### PR TITLE
Add catalog url without updating all the gems I did last time

### PR DESCRIPTION
**Pull request by halloffame**
Last time I ran `bundle update` and Travis kept failing because of that. Hopefully this will fix that.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ gem 'ace-rails-ap'
 gem 'biola_deploy'
 gem 'biola_frontend_toolkit'
 gem 'blazing'
-gem 'buweb_content_models', '~> 0.52.0'
+gem 'buweb_content_models', '~> 0.68.0'
```
Comment (mikestephens) [inline Gemfile]: is there a reason this isnt 0.75.0? (the latest one)

Comment (halloffame) [inline Gemfile]: yeah, that was the latest version on December 5th when I created this PR. I'll update it again in the next PR.

Comment (mikestephens) [inline Gemfile]: oh wow this pr is that old?

**Comment by mikestephens** [comment]
+1

**Pull request merged**